pywr-core/src/test_utils.rs

```diff
@@ -14,20 +14,28 @@ use crate::solvers::ClpSolver;
 use crate::solvers::HighsSolver;
 #[cfg(feature = "ipm-simd")]
 use crate::solvers::SimdIpmF64Solver;
-use crate::timestep::{TimeDomain, Timestepper};
+use crate::timestep::{TimeDomain, TimestepDuration, Timestepper};
 use crate::PywrError;
+use chrono::{Days, NaiveDate};
 use ndarray::{Array, Array2};
 use rand::Rng;
 use rand_distr::{Distribution, Normal};
-use time::ext::NumericalDuration;
-use time::macros::date;
 
 pub fn default_timestepper() -> Timestepper {
-    Timestepper::new(date!(2020 - 01 - 01), date!(2020 - 01 - 15), 1)
+    let start = NaiveDate::from_ymd_opt(2020, 1, 1)
+        .unwrap()
+        .and_hms_opt(0, 0, 0)
+        .unwrap();
+    let end = NaiveDate::from_ymd_opt(2020, 1, 15)
+        .unwrap()
+        .and_hms_opt(0, 0, 0)
+        .unwrap();
+    let duration = TimestepDuration::Days(1);
+    Timestepper::new(start, end, duration)
 }
 
 pub fn default_time_domain() -> TimeDomain {
-    default_timestepper().into()
+    TimeDomain::try_from(default_timestepper()).unwrap()
```

Comment:
`default_timestepper().try_into().unwrap()` should also work here and not require the `TimeDomain` type to written twice.